buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -26,7 +26,7 @@ object Plugins {
   const val kotlinParcelize = "kotlin-parcelize"
   const val androidLibrary = "com.android.library"
   const val androidApplication = "com.android.application"
-  const val hilt = "dagger.hilt.android.plugin"
```

Comment:
The Hilt Gradle Plugin now supports the updated Gradle plugins DSL as described in https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block via its new id: com.google.dagger.hilt.android. (https://github.com/google/dagger/commit/e50aa067efc6484e165e6e0715dc7391487a03f1)